exercises/binary-search/binary_search_tests.plt

```diff
@@ -0,0 +1,33 @@
+:- begin_tests(binary_search).
+
+    test(finds_a_value_in_an_array_with_one_element) :-
```

Comment:
This always gets me when switching languages. We don't use "arrays" in Prolog. Change it to "list" in the test names please?